src/gdcdictionary/schemas/treatment.yaml

```diff
@@ -7176,6 +7200,9 @@ properties:
       - Other
       - Unknown
       - Not Reported
+    deprecated_enum:
+      - Brain Stem
+      - Common Duct
```

Comment:
I can't find the direction to deprecate these enums in [DICT-516](https://gdc-ctds.atlassian.net/browse/DICT-516)

Reply:
oh weird. I saw in the jira for sample that we are supposed to deprecate these enums, but it's not in the sample PR.

Reply:
Yep, looks like the changes took place for wrong prop. Fixed.